.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
maybe add this folder to the .gitignore file

Reply:
Added the folder to the .gitignore file